api-specificatie/openapi.yaml

```diff
@@ -117,13 +117,7 @@ paths:
       operationId: raadpleegAdresMetZoekResultaatIdentificatie
       tags:
         - Adres
-      summary: De API levert huidige gegevens van een adres.
-      description: "Raadpleeg het adres met de identificatie van het gekozen item verkregen via de zoek-operatie.
-        Geleverd worden de huidige gegevens van een adres evt. met embedded woonplaats, openbare ruimte, nummeraanduiding.
-        Huidige gegevens zijn actuele gegevens die geen eind status hebben, zoals 'Naamgeving ingetrokken'.
-        Als de nummeraanduiding is ingetrokken, dan levert een bevraging geen adres op.
-        Relaties met adresseerbaarobject (ligplaats, standplaats of verblijfsobject) en als het adresseerbaare object een verblijfsobject betreft het/de gerelateerde pand(en).
-        Parameter zoekresultaatidentificatie bevat identificatie van een gekozen item verkregen uit een zoek-operatie."
+      description: "Vind een adres met de identificatie van het zoekresultaat uit zoek-operatie get/adressen/zoek."
```

Comment:
@JohanBoer Deze description moet nog worden aangepast n.a.v. de wijzigingen in #137. Daar worden er meer parameters toegevoegd.

Reply:
Ook hiervoor geldt dat Cathy de uiteindelijke descriptions aanlevert.